docs/azure/aksextensions.md

```diff
@@ -0,0 +1,127 @@
+
+2. Click on **Diagnostic Settings** under **Monitoring** in the left navigation.
+
+3. Click on **Add diagnostic setting**.
+
+4. Enter a name, such as myAKSClusterLogs, then select the option to **Archive to a storage account**.
+
+5. Select a storage account of your choice.
+
+6. In the list of available logs, select the logs you wish to enable.
+    > Note: The incurred cost is subject to your storage account usage and Azure Storage Policy.
+
+7. When ready, select **Save** to enable collection of the selected logs.
+
+For more information on Diagnostics settings, visit [Create diagnostic settings to send platform logs and metics to different destinations](https://docs.microsoft.com/azure/azure-monitor/platform/diagnostic-settings).
+
+### Install Azure Service Operator
+
+Right click on your AKS cluster and click on **Install Azure Service Operator** to easily deploy the latest version of Azure Service Operator (ASO) on your AKS cluster and provision and connect applications to Azure resources within Kubernetes. When you select this option, you'll be prompted for a service principal for ASO to use when performing Azure resource operations. This service principal must have appropriate permissions (typically Contributor at suitable scope). Fill out the service principal details and click **Submit** to kick off the installation of Azure Service Operator.
+
+> Install Azure Service Operator can only be performed on an AKS cluster that has never had ASO installed before. If you have already initiated the installation manually, follow the instructions on [Azure Service Operator](https://azure.github.io/azure-service-operator/#installation) to complete.
+
+For more information on Azure Service Operator, visit [Azure Service Operator (for Kubernetes)](https://github.com/Azure/azure-service-operator). If you are experiencing issues with Azure Service Operator, visit [Azure Service Operator (ASO) troubleshooting](https://github.com/Azure/azure-service-operator/blob/master/docs/troubleshooting.md).
+
+![Azure Service Operator Webview](images/aksextensions/azure-service-operator-screenshot.png)
+
+### Create GitHub Workflow
+
+Right click on your AKS cluster and click on **Create GitHub Workflow** to easily open and create a workflow starter template. This helps in quick generation of the workflow templates with pre populates resource and clustername for:
+
+- [Starter Workflow](https://github.com/actions/starter-workflows/blob/main/deployments/azure-kubernetes-service.yml)
+- [Helm Workflow](https://github.com/actions/starter-workflows/blob/main/deployments/azure-kubernetes-service-helm.yml)
+- [Kompose Workflow](https://github.com/actions/starter-workflows/blob/main/deployments/azure-kubernetes-service-kompose.yml)
+- [Kustomize Workflow](https://github.com/actions/starter-workflows/blob/main/deployments/azure-kubernetes-service-kustomize.yml)
+
+
+Congratulations! Now your app is successfully running in Azure Kubernetes Service!
```

Comment:
Is this bit left over from a copy-paste? The doc wasn't really about creating and running an app, was it?